tests/test_procedure.py

```diff
@@ -175,9 +175,8 @@ def test_treatment_started(self):
         for code, display, system in started_codes:
             self.add_procedure(code, display, system)
             self.test_user = db.session.merge(self.test_user)
-            assert (known_treatment_started(self.test_user),
-                    "treatment {} didn't show as started".format(
-                        (system, code)))
+            assert known_treatment_started(self.test_user),\
+                "treatment {} didn't show as started".format((system, code))
```

Comment:
What's the function of this line? I don't think it should be part of the `assert`...

Reply:
assert can take a second parameter, displayed when it fails to help us figure out the failure.

When enclosing it in parens, it will never fail with the second parameter.